README.rst

```diff
@@ -9,5 +9,4 @@ To run::
 	$ virtualenv -p python3 /var/tmp/deckhand
```

Comment:
It looks like you adopted the pattern I used in Drydock. In the near future we should move all the UCP services to demonstrate usage based on containers. This offers the same isolation as virtualenvs and is closer to how they will be used in production. As mark-burnett said, not specific to this PR and can be addressed in the future.